rspotify-model/src/show.rs

```diff
@@ -136,7 +135,6 @@ pub struct FullEpisode {
     pub show: SimplifiedShow,
     #[serde(rename = "type")]
     pub _type: String,
```

Comment:
Same here

Reply:
Tried it in the console and it seems we can remove it, as its value is `"episode"`: https://developer.spotify.com/console/get-episode/?id=512ojhOuo1ktJprKbVcKyQ&market=